.github/.wordlist.txt

```diff
@@ -399,6 +401,7 @@ df
 dfe
 dfu
 DgDxsfHx
+DTS
```

Comment:
is DTS a word? same with BLDevCube?

Generally in markdown you can escape using backticks. We do this for codewords or for shell commands

Reply:
DTS/BLDevCube/PSRAM removed from .wordlist.txt and using backticks to escape these words.